src/main/java/net/mcreator/preferences/data/BlocklySection.java

```diff
@@ -27,6 +27,8 @@
 public class BlocklySection extends PreferencesSection {
 
 	public StringEntry blockRenderer;
+	public IntegerEntry colorSaturation;
+	public IntegerEntry colorValue;

```

Comment:
Given HSB and HSV usually are about the same color model, I think you could name this `colorBrightness` to avoid (a little) too common names. Or is it already used somewhere?

Reply:
HSL and HSV are not the same color model. Blockly uses HSV so is correct parameter is called value. We only call it brightness in UI because it will be clearer to the end-users who are usually not aware of the conical differences between the two.

---------

Generally, community reviews are more needed in terms of testing the PR thoughtfully and reporting if everything works fine and if any bugs were found more than comments about indentation and variable naming that will be handled by maintainers later if needed.